README.md

```diff
@@ -147,6 +147,17 @@ spec:
 
 You can add a prefix to volume display names by setting an `OCI_VOLUME_NAME_PREFIX` environment variable.
 
+### Rounding up volumes less than 50GB
+
+Currently OCI does not support block volumes smaller than 50GB. By default the volume provisioner will round up PVC request for volumes smaller than
```

Comment:
@owainlewis I strongly believe rounding should be opt-in rather than opt-out. Is there a requirement driving this?

Reply:
This needs updating to reflect the reversal of the default (presuming we go ahead with it).

Reply:
@prydie the requirement is for this to be "on" by default for managed K8s. Consideration is around ease of update (i.e rolling out this change to existing clusters and not having to update configuration for all of them).